RecoTracker/IterativeTracking/python/TobTecStep_cff.py

```diff
@@ -109,10 +109,29 @@
         _StripSubClusterShapeSeedFilter.clone()
     )
 )
+_tobTecStepSeedComparitorNoSubClusterFilterPSet = dict(
+    ComponentName = 'CombinedSeedComparitor',
+    mode          = cms.string('and'),
+    comparitors   = cms.VPSet(
+        cms.PSet(# FIXME: is this defined in any cfi that could be imported instead of copy-paste?
+            ComponentName      = cms.string('PixelClusterShapeSeedComparitor'),
+            FilterAtHelixStage = cms.bool(True),
+            FilterPixelHits    = cms.bool(False),
+            FilterStripHits    = cms.bool(True),
+            ClusterShapeHitFilterName = cms.string('tobTecStepClusterShapeHitFilter'),
+            ClusterShapeCacheSrc = cms.InputTag('siPixelClusterShapeCache') # not really needed here since FilterPixelHits=False
+        ),
+    )
+)
+
 tobTecStepSeedsTripl = _seedCreatorFromRegionConsecutiveHitsTripletOnlyEDProducer.clone(#empirically better than 'SeedFromConsecutiveHitsTripletOnlyCreator'
     seedingHitSets     = 'tobTecStepHitTripletsTripl',
     SeedComparitorPSet = _tobTecStepSeedComparitorPSet,
 )
+
+from Configuration.ProcessModifiers.approximatedClusters_cff import approximatedClusters
+approximatedClusters.toModify(tobTecStepSeedsTripl,SeedComparitorPSet = _tobTecStepSeedComparitorNoSubClusterFilterPSet)
```

Comment:
can this just remove `SeedComparitorPSet.comparitors[1]` without a copy-pasted temporary?